tests/test_copyright.py

```diff
@@ -20,6 +20,8 @@
 import re
 import unittest
 
+RE_YEAR = re.compile(r"")
```

Comment:
This is not used and not needed.

Reply:
Woopsie daisy, forgot about that one there...